opengrok-indexer/src/test/java/org/opengrok/indexer/history/GitTagParserTest.java

```diff
@@ -0,0 +1,233 @@
+            "131a4996283810d2ad2cdf42ca5cf0aeb3c69f5b:1493821277:tag: 1.1-rc1:\n" +
+            "2850968932c2933b8ebb1f9808991cda0be846ed:1491038723:tag: 1.0:\n" +
+            "33d7d01d2dbe72c5d9ccc431c31259ba33706f26:1487945373:tag: 0.13-rc10:\n" +
+            "1c66474546b3ca27b8803967e33e9c6d9846f063:1486474858:tag: 0.13-rc9:\n" +
+            "741c4ac5d385192d694de45e7da3e6d23b0b0bd1:1485598510:tag: 0.13-rc8:\n" +
+            "0be0dfb9fbfa472b031826520354de7a4e178fea:1485263221:tag: 0.13-rc7:\n" +
+            "a92452fdda1dc92bfe0edb969f22a290bc933768:1484741212:tag: 0.13.rc6:\n" +
+            "b6edf1ff5c2a6c8755bb50637c323712df1702c5:1480942653:tag: 0.13-rc5:\n" +
+            "f8419a9bdcfa4f940bfac08a82a0cf1289ed93d0:1476781473:tag: 0.13.rc4:\n" +
+            "c7119d5952a97effbc6b5a9779cd1f2bc3e1192e:1476101665:tag: 0.13.rc3:\n" +
+            "506e0a1ddf50341a0603af27ecc254ccb72d7dcb:1473681887:tag: 0.12.1.6, origin/0.12-stable:\n" +
+            "d305482d0acf552ccd290d6133a52547b8da16be:1427209918:tag: 0.12.1.5:\n" +
+            "82e26a688bcd4652544411641c43a5c550d355bf:1422008001:tag: 0.12.1.4:\n" +
+            "a999dfc7134de6a0223141aaf5923225a821c7f2:1421423948:tag: 0.12.1.3:\n" +
+            "815f08b57f69912f6fe9afe3c91e843a37a8adf5:1421316426:tag: 0.12.1.2:\n" +
+            "4b67e8aa5d41fa20f4bab248f1ff04df94f40b8f:1402650166:tag: 0.12.1.1:\n" +
+            "e0d03478c01015c1350e11662b5004cdf34ca58b:1468940162:tag: 0.13-rc2:\n" +
+            "a36364e104e7472e0b93db94a6d8fa458f005d18:1465375689:tag: 0.13-rc1:\n" +
+            "46014450c7a107b2f10690b17559958ddb79f960:1398769525:tag: 0.12.1:\n" +
+            "86135a28ef6bba1816deeab0b5a65bd2349afa15:1396963051:tag: 0.12:\n" +
+            "653794abf1d9fda5f111e2401d8bd3ead80cfc83:1394030192:tag: 0.12-rc7:\n" +
+            "3e0be6bdaa239e1f9f6a95055faf520f2e7c7a86:1388762887:tag: 0.12-rc6:\n" +
+            "5900e47e75dd31beacbcfbe39538550233625ece:1387555815:tag: 0.12-rc5:\n" +
+            "6ca97f885c9c11acf9a3319b9e87ca3d20bc120a:1386937529:tag: 0.12-rc4:\n" +
+            "6bafe4d14b0eb50c353aa9b25365e33b81938328:1386164499:tag: 0.12-rc3:\n" +
+            "61172cb46e108f65a356772860ac577141fdf8e8:1385651730:tag: untagged-2d067cc3eab919a1b8d1:\n" +
+            "17d2cbd8d550eac92cd13955d63de4298303e4e0:1382628329:tag: 0.12-rc2:\n" +
+            "c6963a7ea2753672325502d342e653700be550a8:1377876557:tag: 0.12-rc1:\n" +
+            "c23e82b612acd5e947c164114377578116f6d298:1163621273::";
+
+    @Test
+    public void shouldParseOpenGrokTagsLog() throws IOException {
+        final byte[] OGK_LOG_BYTES = OGK_LOG.getBytes(StandardCharsets.UTF_8);
+        ByteArrayInputStream bytesIn = new ByteArrayInputStream(OGK_LOG_BYTES);
+
+        TreeSet<TagEntry> tagList = new TreeSet<>();
+        GitTagParser parser = new GitTagParser(tagList);
+        parser.processStream(bytesIn);
+
+        /*
+         * The following comparison works because there is no trailing space in
```

Comment:
nit: the real output from `git log --tags ..` has newline at the end

Reply:
Oh you're right